README.md

```diff
@@ -841,6 +841,61 @@ Going crazy:
 ; => {:x 24}
 ```
 
+## To and from JSON
+
+The `m/encode` and `m/decode` functions work on clojure data. To go
+from clojure data to JSON, you need a JSON library like
+[jsonista](https://github.com/metosin/jsonista). Additionally, since
+`m/decode` doesn't check the schema, you need to run `m/validate` (or
+`m/explain`) if you want to make sure your data conforms to your
+schema.
+
+To JSON:
+
+```clj
+(def Tags
+  [:map
+   {:closed true}
+   [:tags [:set :keyword]]])
+(jsonista.core/write-value-as-string
+ (m/encode Tags
+           {:tags #{:bar :quux}}
+           mt/json-transformer))
+; => "{\"tags\":[\"bar\",\"quux\"]}"
+```
+
+From JSON without validation:
+
+```clj
+(m/decode Tags
+          (jsonista.core/read-value "{\"tags\":[\"bar\",[\"quux\"]]}"
+                                    jsonista.core/keyword-keys-object-mapper)
+          mt/json-transformer)
+; => {:foo #{:bar ["quux"]}}
```

Comment:
Here `:foo` but in goes `"tags"`?

Reply:
thanks, I switched from a nonsemantic to a semantic example at some point and forgot to update this